README.md

```diff
@@ -1 +1,149 @@
-# book_store
\ No newline at end of file
+# book_store
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📚 Book Store 
+
+This is an MVP version of awesome_books that allows users to "Display a list of books", "Add a book". and "Remove a selected book".
+ 
+
+## 🛠 Built With <a name="built-with"></a>
+
+<!-- Tech Stack -->
+### Tech Stack <a name="tech-stack"></a>
+
+**React**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link]()
```

Comment:
- [ ] Since you currently do not have a demo link, kindly replace the broken link with a text like ```coming soon```. This would make your documentation more professional

Reply:
Changes implemented. Thank you